board-guide/readme.md

```diff
@@ -1,46 +1,30 @@
-> **Goal of this guide:** Explaining how the functionality of the doable board works and how to create a good user story.
+> *Goal of this guide:* Explain how to work with the achievable board in the interface and with achievables.
 
-# How to use the Achievable board
+# How to use the achievable board
 
-<img src="../images/board.png" width="700">
+The main tool in our process is the achievable board. The board is the place where we keep track of our ideas, where they are in their lifecycle and who is working on what.
 
-The Board is part of [The Interface](https://interface.newatoms.com/). Here is where we can see what needs to be done. We can add *cards* which always contain a *user story*. They're part of our Agile way of working.
+![The board in action](../images/board.png)
 
-## User Story
+## The life of an achievable
 
-A user story is a description consisting one sentence and is chopped up in three parts. It describes a task that has to be done. Never make big user stories, always try to cut a problem into the smallest user stories that still make sense.
+From creation to going live [achievables](../glossary/achievable.md) go through distinct phases, like growing up. On the board there is one column for every [phase](../glossary/phase.md) an achievable is in.
 
-* Goal (To)
-* Person (as a)
-* Action (I can)
+Through its life an achievable travels from left to right.
```

Comment:
Maybe explaning that an achievable can be seen as a physical card to make it more tangible.